zen-quotes-app/src/app/app.module.ts

```diff
@@ -3,15 +3,45 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { AuthorsComponent } from './authors/authors.component';
+import { RouterModule } from "@angular/router";
+import { HomeComponent } from './home/home.component';
+import { BalanceComponent } from './balance/balance.component';
+import { SidebarNavComponent } from './sidebar-nav/sidebar-nav.component';
+import { HttpClientModule } from '@angular/common/http';
+import { MyQuotesComponent } from './my-quotes/my-quotes.component';
+import { LoginComponent } from './login/login.component';
+import { QuotesComponent } from './quotes/quotes.component';
+import { RandomQuoteComponent } from './random-quote/random-quote.component';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { FormsModule } from "@angular/forms";
 
 @NgModule({
   declarations: [
-    AppComponent
-  ],
-  imports: [
-    BrowserModule,
-    AppRoutingModule
+    AppComponent,
+    AuthorsComponent,
+    HomeComponent,
+    BalanceComponent,
+    SidebarNavComponent,
+    MyQuotesComponent,
+    LoginComponent,
+    QuotesComponent,
+    RandomQuoteComponent
   ],
+    imports: [
+        BrowserModule,
+        AppRoutingModule,
+        HttpClientModule,
+        RouterModule.forRoot([
```

Comment:
Q: Is this needed if you already import `AppRoutingModule`?

Thanks! 🙏 🙇

Reply:
Thank you very much! Deleted 🙏

Reply:
Wonderful, thank you! 🙏

Reply:
Thank you! 🙏